src/pfe/portal/routes/projects/remoteBind.route.js

```diff
@@ -142,6 +146,7 @@ async function bindStart(req, res) {
       error: err.info || err
     }
     user.uiSocket.emit('projectBind', data);
+    res.status(500).send(`Project creation failed for project ${data.name}. Error: ${data.error}`);
```

Comment:
This is called after we will have finished with the `res` object and sent `202`. Here the response can only be reported by the socket.